redhat/rdma-core.spec

```diff
@@ -43,6 +43,7 @@ BuildRequires: perl-generators
 %endif
 
 Requires: dracut, kmod, systemd, pciutils
+Requires: libibverbs-utils%{?_isa} = %{version}-%{release}
```

Comment:
Maybe we should move it to libibverbs-utils, or possibly just delete it? rxe_cfg horrible, broken and obsolete.

Reply:
> Maybe we should move it to libibverbs-utils, or possibly just delete it? rxe_cfg horrible, broken and obsolete.

We can't delete rxe_cfg, because it already get in rhel distros. Redhat has to support as it is in support life cycle. So, move it into libibverbs-utils looks a better choice.

Reply:
I was under impression that RHEL manages its own version of spec and rdma-core. For example, what did you do with removed drivers (nes, cxgb3 e.t.c)?

Reply:
Yes, you are right. We (Redhat) do have own version spec files for RHEL.

We also try our best to decrease the difference between upstream spec file with
Fedora/RHEL specific spec file. 

1) When there is some thing wrong with Fedora/RHEL rdma-core, I always test the
upstream version of rdma-core. As I know, most bugs are not Fedora/RHEL specific,
that means such bugs also impacted upstream rdma-core. The difference between 
upstream and Fedora/RHEL is really annoy, as I need pay extra attention to handle
them, when building/debugging with upstream rdma-core.

2) Redhat always build latest upstream release rdma-core for Fedora Rawhide, the
difference between upstream and Fedora may introduce regression issue because of
oversight when building rpm for Fedora Rawhide. Again, the difference may introduce
regression issue again, when backport Fedora version rdma-core for RHEL major release.

Reply:
I thought rxe was tech preview at best in RHEL? Why is there a problem to delete rxe_cfg for tech preview suff?

Reply:
The actual historical path was that we took rxe to non-tech preview for one release cycle in rhel7 and then reversed that when we saw how many problems it had.  It is now tech preview in the latest rhel7 and rhel8.  Because of that, we can probably justify removing rxe_cfg and forcing an update of the iproute2 package to get the latest rdma tool, but we would need to identify a version of iproute2 that is "feature complete" as far as managing rxe devices is concerned, and then should ideally also create some systemd templates for autostarting rxe on top of tagged net interfaces.